work/v1/types.go

```diff
@@ -545,6 +545,13 @@ type AppliedManifestWorkStatus struct {
 	// However, the resource will not be undeleted, so it can be removed from this list and eventual consistency is preserved.
 	// +optional
 	AppliedResources []AppliedManifestResourceMeta `json:"appliedResources,omitempty"`
+
+	// EvictionStartTime represents the current appliedmanifestwork will be evicted after a grace period.
+	// An appliedmanifestwork will be evicted from the managed cluster in the following two scenarios:
+	//   - the manifestwork of the current appliedmanifestwork is missing on the hub, or
+	//   - the appliedmanifestwork hub hash does not match the current appliedmanifestwork hub hash.
```

Comment:
the appliedmanifestwork hub hash does not match the current hub hash of the work agent.

Reply:
updated